scripts/build.ts

```diff
@@ -0,0 +1,76 @@
+import { pick } from '@hyperlane-xyz/utils';
+import fs from 'fs';
+import { parse } from 'yaml';
+import { CoreChains } from '../chains/core';
+
+function genJsExport(data, exportName) {
+  return `export const ${exportName} = ${JSON.stringify(data, null, 2)};`;
+}
```

Comment:
is this a common pattern? kinda surprised that we are exporting non-static assets tbh

Reply:
how useful is this?